panel/src/components/Forms/Blocks/Blocks.vue

```diff
@@ -595,6 +595,15 @@ export default {
       this.$emit("input", this.blocks);
     },
     select(block, event = null) {
+      // resets multi selecting on tab change
+      // keep only if there are already multiple selections
+      // triggers `blur` event when tab changed
+      window.addEventListener('blur', () => {
+        if (this.batch.length === 0) {
+          this.isMultiSelectKey = false;
+        }
+      });
+
```

Comment:
Isn't `select()` called on blur anyways already but as `select(null)`? I think we should try to rather work with that.

Actually the following comment already addresses the referenced issue:
```
// checks the event just before selecting the block
// especially since keyup doesn't trigger in with
// `ctrl/alt/cmd + tab` or `ctrl/alt/cmd + click` combinations
// for ex: clicking outside of webpage or another browser tab
if (event && this.isMultiSelectKey) {
```

So to me it seems this comment is actually wrong. Seems like `⌘⌥→` DOES trigger the `keyup` event - or at least on the `keydown` event. We actually listen to both. Maybe that's also the issue.

Or we can inspect the `event` parameter and from that determine which calls of select is actually happening.

Just adding another listener to blur (which could be any blur, not just another tab) - not sure if that feels right.

Reply:
> Isn't select() called on blur anyways already but as select(null)? I think we should try to rather work with that.

I know the `select(null)`, already tried. `blur` not triggered when browser tab changed.

> Actually the following comment already addresses the referenced issue:

According to the description yes, but those lines only fix the click issue with browser tab changes.

As a result, this new change solves both the tab and click issue. We can remove the codes you mentioned.